config/lava/base/kernel-ci-base.jinja2

```diff
@@ -64,6 +64,10 @@ timeouts:
   actions:
     power-off:
       seconds: 30
+{%- if queue_timeout %}
+  queue:
+    {{ queue_timeout }}
+{% endif %}
```

Comment:
If you want to control how the YAML is going to be generated then I believe you have to list each possible field from the dictionary, like:
```
{%- if queue_timeout.hours %}
        hours: {{ queue_timeout.hours }}
{% endif %}
```
which isn't great, but at least it explicitly makes it clear how this should be in YAML.  The current implementation just works because the Python string representation of a dictionary is compatible with YAML, but that's mostly luck.  Maybe it could be done by iterating over the key/value pairs of the `queue_timeout` variable to avoid repeating the hours / minutes etc.

Reply:
Doing this could be a way to validate content of queue_timeout (we want basicly only days or hours)

Reply:
Right, well the validation could be done separately from generating a definition. There's already `kci_test validate` for that, basically the code that loads the config should be validating it.

Ideally we should be using something like `jsonschema` to describe the YAML config schema but there are other things to fix first.